services/client/src/pages/developers/index.js

```diff
@@ -0,0 +1,65 @@
+import React, { Component } from "react";
+import axios from "axios";
+import _ from "lodash";
+import moment from "moment";
+import CallToActionBanner from "components/call-to-action-banner";
+import Spinner from "components/spinner/loading";
+import Developer from "components/developer";
+
+const updateDevelopersList = function(developersList) {
+  const developers = _.map(developersList, item =>
+    _.extend({}, item, { created: moment(item.created).valueOf() })
+  );
+  return _.orderBy(developers, ["repositories"], ["desc"]);
```

Comment:
`_.shuffle(developers)` :trollface:

Reply:
Turns out people who have less public repos and gists also have missing images, bios, companies, blog,  etc but having a way to sort this and filter this will be useful in the future.